examples/qm9/train_qm9.py

```diff
@@ -124,7 +124,8 @@ def main():
         print('preprocessing dataset...')
         preprocessor = preprocess_method_dict[method]()
         dataset = D.get_qm9(preprocessor, labels=labels)
-        os.makedirs(cache_dir)
+        if not os.path.exists(cache_dir):
+            os.mkdir(cache_dir)
```

Comment:
Use `makedirs` instead of `mkdir`, as we need to create directories recursively (e.g . `input/nfp_NR-AR`).